polar-core/src/terms.rs

```diff
@@ -311,6 +331,14 @@ impl Term {
         }
     }
 
+    pub fn offset_to_end(&self) -> usize {
```

Comment:
Reminder to delete this code when error context is propagated from the parser correctly.